logdetective/server/gitlab.py

```diff
@@ -118,7 +118,7 @@ def is_eligible_package(project_name: str):
     # First check the allow-list. If it's not allowed, we deny.
     allowed = False
     for pattern in SERVER_CONFIG.general.packages:
-        print(f"include {pattern}")
+        LOG.debug("include %s", pattern)
```

Comment:
That was actually supposed to be removed entirely from my original patch. I missed it.